### PR TITLE
Remove support of default field + do not explode CSV string/value

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

File: src/Model.php [hunk 1]
```diff
@@ -1898,23 +1881,10 @@ protected function _rawInsert($m, $row)
...
 
-        if (!is_array($row)) {
-            $row = [$m->title_field => $row];
-        }
-
         // Find any row values that do not correspond to fields, and they may correspond to
         // references instead
         $refs = [];
         foreach ($row as $key => $value) {
-            // no field exists
```
Comment (DarkSide666) [inline src/Model.php]: I don't see reason to remove this block. It's not related to default title_field or CSV and it's actually used in atk4\data\Reference\HasMany.

File: tests/LimitOrderTest.php [hunk 2]
```diff
@@ -46,7 +46,6 @@ public function testReverse()
         $ii->addExpression('total_gross', '[total_net]+[total_vat]');
         $ii->getField($ii->id_field)->system = false;
 
-        // pass parameters as CSV string
```
Comment (DarkSide666) [inline tests/LimitOrderTest.php:49]: this line belongs to CSV in setOrder :)

**Review by DarkSide666 — CHANGES_REQUESTED** [review]

File: tests/LimitOrderTest.php (hunk 2 above)
Comment (mvorisek) [reply tests/LimitOrderTest.php:49]: True, but very unclear comment, comments like this are bad - code change a comments very often do not.

File: src/Model.php (hunk 1 above)
Comment (mvorisek) [reply src/Model.php]: It is actually related with CSV - see the explode there :)

**Review by DarkSide666 — APPROVED** [review]

**Pull request merged**